ui.R

```diff
@@ -737,12 +737,19 @@ tabPanel("User Guide",
                             downloadButton("UG", "Download User Guide"),
                             br(),
                             br(),
-                            h4 (tags$strong("Cochrane training Webinar materials", style = "cursive;
+         h4 (tags$strong("ESMARConf 2023 Tutorial", style = "cursive; color: #2196c4")),
```

Comment:
`style = "cursive;"` is not a valid css styling, for cursive text this should be `style = "font-family: cursive; color: #2196c4"`

Reply:
I've removed the instances of this from the tab (was copied and pasted from existing code)